### ISSUE TITLE
Serve the autocomplete css (algolia-autocomplete.css) minified

### ISSUE THREAD

**Issue by andizer** [algolia/algoliasearch-wordpress#764]
### What did you expect to happen?
The CSS file being served minified.  When served minified, the page will be loaded a little bit faster.

### What happened instead?
The CSS file is not minified.

### How can we reproduce this behaviour?
Open the CSS file served on the frontend and see it is not minified.

Can you provide a link to a page which shows this issue?
N/A

### Technical info
* WordPress version: 4.9.6
* Algolia Search plugin version: 2.11.1

**Comment by rayrutjes** [comment]
Indeed, the file is not minified: https://github.com/algolia/algoliasearch-wordpress/blob/master/includes/class-algolia-plugin.php#L155

We would need to add that to the build process.

Feel free to submit a PR. Happy to move this forward.
